andela/tsunade-cfh
# #137090909 Setup App with Jasmine and Karma

#### What does this PR do?
This PR setup the existing application with Jasmine and Karma 
#### Description of Task to be completed?
To ensure the front end has all functionality work the way it was intended. Use Jasmine and Karma(with PhantomJS, Chrome and Firefox) to setup front end testing environment.
#### How should this be manually tested?
This is tested by running a cloned and installed copy of the repo and installing all the dependencies for jasmine and karma and running the test(s) to see them pass and fail.
#### Any background context you want to provide?
No
#### What are the relevant pivotal tracker stories?
Setup unit testing for the front end
#### Screenshots (if appropriate)
<img width="1280" alt="test_shot" src="https://cloud.githubusercontent.com/assets/24937779/21859301/5ad3c6aa-d82c-11e6-95b7-0975978281ea.png">

#### Questions:
No

#### Adding your test files
- Add your test(s) file to unit directory inside the test folder i.e. test/unit/yourtest.js
- For Karma to know where to run your test(s), go to the karma.conf.js  in the root directory and add your test file(s) under the files: [ ] block

- Run Tests for the application with: 

```
karma start
```
